### PR TITLE
ATS-770: Tech Debt: Transforms - Tiff (via ImageMagick) - add unit tests for supported transforms

### PR DESCRIPTION
**Pull request by montgolfiere**
- add unit test for image/tiff -> image/*
- note: some require request to contain transform options (startPage) for single page output

### CODE REVIEW

File: alfresco-transform-imagemagick/alfresco-transform-imagemagick-boot/src/test/java/org/alfresco/transformer/ImageMagickTransformationIT.java [hunk 1]
```diff
@@ -217,8 +231,16 @@ public void testTransformation()
             sourceFile, sourceMimetype, targetMimetype, targetExtension);
         try
         {
+            // note: some image/tiff->image/* will return multiple page results (hence error) unless options specified for single page
+            Map<String, String> tOptions = emptyMap();
+            Pair targetPair = Pair.of(targetExtension, targetMimetype);
+            if (targetExtensionsForTiffFirstPage.contains(targetPair))
```
Comment (KikoUK) [inline alfresco-transform-imagemagick/alfresco-transform-imagemagick-boot/src/test/java/org/alfresco/transformer/ImageMagickTransformationIT.java:237]: From a quick look, now we add those options to all types of source files, not just .tiff?

Comment (montgolfiere) [reply alfresco-transform-imagemagick/alfresco-transform-imagemagick-boot/src/test/java/org/alfresco/transformer/ImageMagickTransformationIT.java:237]: Ah, yes - nice catch. I'lll make it more explicit for this special case.

Comment (montgolfiere) [reply alfresco-transform-imagemagick/alfresco-transform-imagemagick-boot/src/test/java/org/alfresco/transformer/ImageMagickTransformationIT.java]: Updated, thanks.

Comment (KikoUK) [reply alfresco-transform-imagemagick/alfresco-transform-imagemagick-boot/src/test/java/org/alfresco/transformer/ImageMagickTransformationIT.java]: LGTM after the update, thanks.

**Pull request merged**